components/Navbar.js

```diff
@@ -0,0 +1,82 @@
+const Navbar = () => (
+  <div className="Navigation">
+    <div className="SoS-Text">
+      <h1>SUMMER OF SHIPPING</h1>
+      <img src="Ship Logo.svg" className="SoS-logo" />
+    </div>
+    <div className="Tabs">
+      <a href="/index">
+        HOME
+      </a>
+
+      <a href="https://forms.gle/3N3CfqbH6YkM1DF67">
```

Comment:
nit: please change this to the `/wiki` page

Reply:
Finished task, change will be present in next commit